samples/nrf5340/smp_svr/README.rst

```diff
@@ -0,0 +1,185 @@
+
+         mcumgr --conntype serial --connstring "/dev/ttyACM2,baud=115200" echo hello
+         hello
+
+.. note::
+   The :file:`mcumgr` command-line tool requires a connection string in order to identify the remote target device.
+   In the Bluetooth version of the sample a Bluetooth LE-based connection string is used, and you may need to modify it depending on the Bluetooth LE controller you are using.
+
+   In the following sections, examples will use ``<connection string>`` to represent
+   the ``--conntype <type>`` and ``--connstring=<string>`` :file:`mcumgr` parameters.
+
+
+Image upload and image mapping to MCUboot slot
+----------------------------------------------
+
+For this sample configuration the mcumgr supports, for upload, 3 target images, of which allows to update all parts of the firmware.
+
+The mcumgr ``image upload`` command may be provided optional ``-e -n <image>`` parameter that will
+select target image for upload; when parameter is no provided, 0 is assumed, which means "default
+behaviour", and it performs upload to the "image-1", the MCUboot secondary slot.
+
+The Partition manager (PM) label for slot-to-``<image>`` translation is shown in the table below:
+
+    +-----------------------+--------+-------------+---------------------------+
+    | PM label              | Slot   | -n <image>  |       Firmware part       |
+    +=======================+========+=============+===========================+
+    | "mcuboot_secondary"   | slot-1 |     0       | internal application part |
+    +-----------------------+--------+-------------+---------------------------+
+    | "mcuboot_secondary_1" | slot-3 |     1       | networking                |
+    +-----------------------+--------+-------------+---------------------------+
+    | "mcuboot_secondary_2" | slot-5 |     2       | qspi application part     |
+    +-----------------------+--------+-------------+---------------------------+
+
+.. note::
+
+   The ``-e`` option means "no erase", and is provided to the Mcumgr to prevent it from sending the erase command to target before updating image.
+   This options is always needed when ``-n`` is used for image selection, as the erase command is hardcoded to erase slot-1 ("image-1"), regardless of which slot is uploaded at the time.
+
+Upload the signed image
+-----------------------
+
+To upload the signed image, use the following command:
+
+.. code-block:: console
+
+   sudo mcumgr <connection string> image upload build/zephyr/<signed_upload.bin> -e -n <image>
+
+Use:
+   `internal_flash_update.bin` file and <image> = 0. for updating the internal application part image.
+   `net_core_app_update.bin` file and <image> = 1. for updating the networking core image
+   `qspi_flash_update.bin` file and <image> = 2. for updating the qspi application part image.
+
+
+.. note::
+
+   The <image> values ``0`` and ``2`` correspond to the internally and the externally linked parts of application, respectively.
```

Comment:
```suggestion
   The ``<image>`` values correspond to the internally (``0``) and externally (``2``) linked parts of the application.
```
@nvlsianpu, @kittydepa was trying something here to make it more straightforward but might've done the opposite - do you think this edit is clear enough? If not then I have no strong feeling when it comes to keeping it